conf/steps/simulation.config

```diff
@@ -40,7 +40,7 @@ process {
         publishDir = [ enabled: false ]
     }
     withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:BAM_DOWNSAMPLE:SAMTOOLS_MERGE' {
-        ext.prefix = { "${meta.id}" }
+        ext.prefix = { "${meta.id}_${meta.depth}x" }
```

Comment:
If you do so, the names of the individuals in the vcf downstream will be set to this prefix.
That's why I've set it to `meta.id` only so that the validation step could work.

Reply:
I see, you're right.